example.js

```diff
@@ -7,21 +7,21 @@ const Instauto = require('instauto'); // eslint-disable-line import/no-unresolve
 const options = {
   cookiesPath: './cookies.json',
 
-  username: 'your-ig-username',
-  password: 'your-ig-password',
+  username: process.env.INSTAGRAM_USERNAME,
+  password: process.env.INSTAGRAM_PASSWORD,
 
   // Global limit that prevents follow or unfollows (total) to exceed this number over a sliding window of one hour:
-  maxFollowsPerHour: 20,
+  maxFollowsPerHour: process.env.MAX_FOLLOWS_PER_HOUR,
```

Comment:
For all these parameters, can we instead default to the old value if environment is not set? Also need to convert most of these to `Number`.
e.g.:
```js
maxFollowsPerHour: process.env.MAX_FOLLOWS_PER_HOUR != null ? parseInt(process.env.MAX_FOLLOWS_PER_HOUR, 10) : 20
```